tempesta_fw/http_parser.c

```diff
@@ -3433,115 +2928,113 @@ tfw_http_parse_req(void *req_data, unsigned char *data, size_t len,
 		/* fall through */
 	}
 
+/*
+ * Fot most (or at least most frequent) methods @step_inc should be
+ * optimized out. The macro is used to reduce the FSM size, so there is not
+ * sense to use it specific versions for the few states, e.g. for 'GET '.
+ */
+#define __MATCH_METH(meth, step_inc)					\
```

Comment:
I think you should not save two letters and use `METHOD`

Reply:
We have 80-characters line limits, so there is no sense to use long names. Any way you understood that `METH` is `METHOD` ;)

Reply:
Perhaps he meant that `METH` is a very popular shortcut for a certain substance. `MTOD` could be another clear shortcut for this without the undesireable connotation.